packages/api/src/target-group/target-groups.service.ts

```diff
@@ -141,6 +139,27 @@ export class TargetGroupsService {
         throw new Error("Brevo Error: Could not create contact list");
     }
 
+    public async createIfNotExistTestTargetGroupForScope(scope: EmailCampaignScopeInterface): Promise<TargetGroupInterface> {
+        const testList = await this.repository.findOne({ scope, isMainList: false, isTestList: true });
+
+        if (testList) {
+            return testList;
+        }
+
+        const title = "Test list for current scope";
```

Comment:
I guess just stringifying the scope would be enough here? @raphaelblum

Reply:
Thank you :)
I didn't think about it

Reply:
I think it would still be better to make it more user-readable. 
But we can think of improving this in another task, since we also should update it for the main list